### DISCUSSION TITLE
Using Bootstrap themes

### DISCUSSION THREAD

**Discussion by gallwood** [Megabit/Blazorise#1809] [Q&A] [answered]
Hi,

I was trying to use a different theme for Bootstrap, for example using this:

`<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootswatch/4.5.2/minty/bootstrap.min.css" integrity="sha384-H4X+4tKc7b8s4GoMrylmy2ssQYpDHoqzPa9aKXbDwPoPUA3Ra8PA5dGzijN+ePnH" crossorigin="anonymous">`

in index.html. This theme is quite a bit different - for my illustrative purposes.

In a normal Blazor app this works (once you've removed a few of the styles in app.css). However, in my project that uses Blazorise, I'm not seeing the new color appear. For example, looking at the default Counter page from the Blazor template.

I was wondering if this is as expected. Is this why Blazorise contains the ThemeProvider?

**Comment by stsrki** [comment, marked as answer]
I tried the theme you used. 

![image](https://user-images.githubusercontent.com/900302/107336488-6f13e180-6ab9-11eb-8f55-eb9fbc07f56d.png)

You probably need to clear the cache by pressing **Ctrl+F5**.

Also, ThemeProvider is used mainly to quickly change theme colors without changing too many other options. So if you don't need it just remove it disable it.

**Reply by gallwood** [reply]
D'oh.

I can't believe I was so stupid. I actually had a ThemeProvider around my Router component. 😄 

At least I now know the ThemeProvider works a treat.

Thanks @stsrki

**Reply by stsrki** [reply]
Np, glad I could help :)
